SAMPLE_FORMAT.md

```diff
@@ -211,21 +374,17 @@ Streams can be extremely compact, efficient, and easy to use - consider using th
 * Checked Exceptions can be problematic inside streams.
   
 ### Parallel Streams
-Parallel Streams make make sense in a few situations. There are many situations where there use is a net loss.  Really think through your usage and consider what they might mean if you are already doing concurrent operations.
+Parallel Streams make make sense in a few situations. There are many situations where there use is a
+net loss.  Really think through your usage and consider what they might mean if you are already
+doing concurrent operations.
 
 ## Additional Best Practices 
-
 The following are some general Java best practices that should be followed in samples to remain
 idiomatic. 
 
-### Style
-Wherever possible (and when not conflicting any of the above guidelines), follow the
-[Google Java Style Guide](https://google.github.io/styleguide/javaguide.html). It's encouraged, but
-not required to use `[google-java-format](https://github.com/google/google-java-format)` to help 
-format your code. 
-
 ### Time
 Use the `java.time` package when dealing with units of time in some manner. 
 
 ### Logging
-Use `java.util.logging` for consistent logging in web applications. 
+Use `java.util.logging` for consistent logging in web applications. Unless you are demonstrating how
```

Comment:
Our [java logging examples](https://cloud.google.com/logging/docs/setup/java#example) tell users to use slf4j. Should we be consistent and start using that be default? (I believe it's compatible with Stackdriver too?)

Reply:
Sg